Makefile

```diff
@@ -110,9 +110,36 @@ deploy-full-local-setup: ## Deploy full local multicluster setup
 	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
 	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
 
+.PHONY: deploy-stable
+deploy-stable:
+	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),)
+	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS))
+
+	$(call list-running-pods,$(CLUSTER_GSLB1))
+	$(call list-running-pods,$(CLUSTER_GSLB2))
+
+.PHONY: deploy-candidate
+deploy-candidate:
+	@echo "\n$(YELLOW)build k8gb docker and push to registry $(NC)"
```

Comment:
It does not really push to registry but importing the image to cluster below, correct?